docs/content/users/usage/commands.md

```diff
@@ -1540,16 +1540,30 @@ ddev typo3 site:show
 
 Print DDEV and component versions.
 
+Flags:
+
+* `--match-constraint <version-constraint>`: Check if the currently installed ddev matches the specified [version constraint](https://github.com/Masterminds/semver#checking-version-constraints).
+
 Example:
 
 ```shell
 # Print DDEV and platform version details
 ddev version
+
+# ddev add-on command was added in v1.23.5, conditionally run the new command
+if ddev version --match-constraint ">= 1.23.5" 2>&1 > /dev/null; then
+  ddev add-on get ....
+else
+  ddev get ....  
+fi
```

Comment:
It looks good, but it won't work as expected with v1.23.5 because there's no `--match-constraint' flag.

I think we need to mention that this flag will run the actual check only from DDEV v1.24.0.

Reply:
@stasadev I improved this, added an extra check and mention about when it was added. I also added it to the flag.

I didn't see any other reference in the docs to when features were introduced, at least not on the commands page.